client/static/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
```

Comment:
`robots.txt` and `favicon.ico` is already in `/static/`... is the plan to remove those in another PR?

Reply:
ah, this is actually because I had merged `main` into this and thus your most recent change came with